bnd.bnd

```diff
@@ -1,2 +1,2 @@
 -includeresource: @org.apache.sling.feature.extension.apiregions-[0-9.]*.jar!/org/apache/sling/feature/extension/apiregions/launcher/LauncherProperties.*
-Conditional-Package: org.apache.sling.feature.extension.apiregions.api
+Conditional-Package: org.apache.sling.feature.extension.apiregions.api,org.apache.felix.utils.manifest
```

Comment:
Can we use the shade plugin to move these classes (the felix ones) to the sling namespace?

Reply:
Actually, while the code for a time used the felix manifest classes, it doesn't now any more, so I have reverted this changed line.